src/sentry/integrations/pipeline.py

```diff
@@ -50,10 +51,27 @@ class IntegrationPipeline(Pipeline):
 
     def finish_pipeline(self):
         data = self.provider.build_integration(self.state.data)
+        try:
+            response = self.reinstall(data)
+        except Integration.DoesNotExist:
+            pass
+        else:
+            return response
         response = self._finish_pipeline(data)
         self.clear_session()
         return response
 
+    def reinstall(self, data):
+        integration = Integration.objects.get(
+            provider=self.provider.key,
+            name=data['name']
+        )
+        org_integration = OrganizationIntegration.objects.get(
+            integration=integration, organization=self.organization)
+        integration.update(external_id=data['external_id'], status=ObjectStatus.VISIBLE)
+        integration.reinstall_repositories()
```

Comment:
Can we rename this function to just `reinstall` and have it on the Integration service object (`get_install`) and then in the GH integration do the instal repo work or something along those lines.

The way this is right now just breaks abstraction a little bit, the integration pipeline probably shouldn't know anything about re-enabling repositories.